src/main/java/com/exasol/sql/ddl/drop/DropTable.java

```diff
@@ -0,0 +1,68 @@
+package com.exasol.sql.ddl.drop;
+
+import com.exasol.sql.*;
+
+/**
+ * This class implements an SQL {@link DropTable} statement
+ */
+public class DropTable extends AbstractFragment implements SqlStatement, DropTableFragment {
+    private final Table table;
+    private boolean ifExists = false;
+    private CascadeConstraints cascadeConstraints = null;
+
+    /**
+     * Create a new instance of an {@link DropTable} statement
+     *
+     * @param tableName name of the table to drop
+     */
+    public DropTable(final String tableName) {
+        super(null);
+        this.table = new Table(this, tableName);
+    }
+
+    @Override
+    public void accept(final DropTableVisitor visitor) {
+        visitor.visit(this);
+        this.table.accept(visitor);
+        if (this.cascadeConstraints != null) {
+            this.cascadeConstraints.accept(visitor);
+        }
+    }
+
+    /**
+     * Add "if exists" expression into a drop table statement
+     *
+     * @return <code>this</code> for fluent Programming
+     */
+    public synchronized DropTable ifExists() {
+        this.ifExists = true;
```

Comment:
This allows chaining multiple `ifExists()`. It's not really a big deal since they all set the argument to `true`.

Reply:
fixed